### PR TITLE
Mark space read improvements

### PR DESCRIPTION
**Pull request by adamweeks**
Added a few things that the web client has. 

* If the window isn't active when the new activity arrives, we won't mark it as read. (previously, if it was scrolled to the bottom, it would auto-ack no matter what the status of the widget/window was)
* Moving the mouse while scrolled to the bottom will mark as read

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview ready!

Built with commit 8d0538e869300472908ea7731f4f7ba2b3515a4f

https://deploy-preview-587--hopeful-hamilton-4090b9.netlify.com

File: packages/node_modules/@ciscospark/widget-space/src/container.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ const injectedPropTypes = {
 export const ownPropTypes = {
   customActivityTypes: PropTypes.object,
   initialActivity: PropTypes.string,
+  isAcknowledgementsDisabled: PropTypes.bool,
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-space/src/container.js]: Should we call this `hasAcknowledgementsDisabled`? I feel it's better grammatically, and signals that "Acknowledgements" inside this component are disabled (e.g. `hasNotificationsMuted`)
I've always considered the `is` prefix to be something specific to this component that doesn't need another noun to tell you what it is (e.g. `isOff`, `isMuted`). What do you think?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-space/src/container.js]: Yea, makes sense!

**Comment by adamweeks** [comment]
Closes #122

**Pull request merged**